RecoParticleFlow/PFProducer/doc/PFProducer.doc

```diff
@@ -1,47 +0,0 @@
-
-/*!
```

Comment:
this will delete doxygen documentation http://cmsdoxygen.web.cern.ch/cmsdoxygen/CMSSW_10_6_0_pre2/doc/html/RecoParticleFlow_PFProducer.html
At least some of the content here is valid.
Please keep (perhaps cleanup).

Reply:
All the better that it's used! Thanks for the explanation. I just brought it back (and rebased everything to not get invalid states).